index.html

```diff
@@ -969,6 +978,49 @@ <h2>
       <p class="issue">TODO add COSE Sign1 detached payload examples</p>
 
     </section>
+    <section>
```

Comment:
```suggestion
    <section class="informative">
```

Reply:
it was not, but it is now: https://github.com/w3c/vc-jwt/pull/114/commits/6bb452f224085e2288a20590f31bac2a8ee05234